source/meshconv/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+project (meshconv CXX)
+
+IF (NOT MR_EMSCRIPTEN)
+  find_package(Boost COMPONENTS program_options REQUIRED )
+  if(Boost_PROGRAM_OPTIONS_FOUND)
+    add_definitions( "-DCGAL_USE_BOOST_PROGRAM_OPTIONS" )
```

Comment:
This line seems unnecessary? @MaxRayskiy

Reply:
Yes